build-system/server/new-server/router.js

```diff
@@ -2,8 +2,20 @@ const router = require('express').Router();
 // @ts-ignore
 const {transform} = require('./transforms/dist/transform');
 
-router.get('/examples/*.html', async (req, res) => {
+// There are some pages that we do not want transformed explicitly.
+const exemptPages = [
```

Comment:
I fear that this list isn't easily discoverable / maintainable and may go out of date when new js-only tests are added. Would it make sense to move these pages into a subdirectory that doesn't match the regex on line 14? (Currently `/examples/*.html`.)

Reply:
would it be ok to go with anything that ends with `nomodule.html` instead will skip the transformations

Reply:
Good idea, I think that'll work!